packages/scripts/src/migration/dvm2-designated-voting/0_Deploy.ts

```diff
@@ -62,6 +59,10 @@ async function main() {
   console.log("Deploying contracts...");
 
   for (let i = 0; i < numberOfPayloadsToBuild; i++) {
+    if (i != 5) {
+      console.log("SKIPPING");
```

Comment:
Sorry, this may be a dumb question, but why are we skipping everything ut the 5th one? Can you add a comment explaining this?

Reply:
this was mistakenly committed. I had run out of eth on the deployment address I was using and needed to re-run part of the deployment to fix the missing bit in the payload :)